packages/twenty-ui/src/theme/constants/ThemeCommon.ts

```diff
@@ -15,14 +16,14 @@ export const THEME_COMMON = {
   animation: ANIMATION,
   spacingMultiplicator: 4,
   spacing: (...args: number[]) =>
-    args.map((multiplicator) => `${multiplicator * 4}px`).join(' '),
-  betweenSiblingsGap: `2px`,
+    args.map((multiplicator) => `${multiplicator * 4}px`).join(' ') as Spacing,
+  betweenSiblingsGap: `2px` as Spacing,
   table: {
-    horizontalCellMargin: '8px',
-    checkboxColumnWidth: '32px',
-    horizontalCellPadding: '8px',
+    horizontalCellMargin: '8px' as Spacing,
```

Comment:
I don't think I've ever seen us doing this. Why do we need it here? Seems like a pattern we want to be careful introducing, if we do it we probably have to do it everywhere

Reply:
It's beneficial to use this type because it ensures that a prop is a Spacing. This doesn't affect the current usage since Spacing is a string. Therefore, if you use a prop of type string, you can provide a Spacing. However, by opting to use a Spacing, as I did in the components, you ensure that the prop will be a spacing.
Additionally, in the Logo component, this is particularly useful because when using the CSS calc function, you need to define the unit in only one of the parameters.

For example:
```
  calc((${({ theme }) => theme.spacing(3)} * -1) * ${({ size }) => size} / 48);
```
This is valid because:
- `theme.spacing(3) * -1`  = -12**px**
- `${({ size }) => size}` = 48
- `48` = 48

Only one parameter returns a unit, so CSS recognizes it as a px spacing. 

With a string type instead of a Spacing type, developers might make mistakes, such as using 2rem. So this kind of improvement should increase the DX and the coherence of the global codebase.

Reply:
Mmhh this is very interesting but I think it requires more work

We should look into 
https://emotion.sh/docs/typescript